opencl-aot/source/main.cpp

```diff
@@ -253,17 +265,46 @@ getCompilerBuildLog(const CLProgramUPtr &ProgramUPtr, cl_device_id Device) {
 
 int main(int Argc, char *Argv[]) {
   // step 0: set command line options
-  cl::opt<std::string> OptInputBinary(
-      cl::Positional, cl::Required,
-      cl::desc("<input SPIR-V or OpenCL program binary>"),
+  cl::opt<std::string> OptInputFile(
+      cl::Positional,
+      cl::desc("<input SPIR-V or OpenCL program binary or OpenCL C source>"),
       cl::value_desc("filename"));
+  cl::opt<std::string> OptOutputElf(
+      "o", cl::init("aot-output.bin"),
+      cl::desc("Specify the output OpenCL program binary filename"),
```

Comment:
Am I correct that during --cmd=compile, we will get OpenCL program object, not binary?